hoomd/md/pytest/test_potential.py

```diff
@@ -745,76 +744,75 @@ def S_r(r, r_cut, r_on):  # noqa: N802 - allow uppercase function name
         denominator = (r_cut**2 - r_on**2)**3
         return numerator / denominator
 
+    def set_distance(simulation, distance):
+        snap = sim.state.get_snapshot()
+        if snap.communicator.rank == 0:
+            snap.particles.position[0] = [0, 0, .1]
+            snap.particles.position[1] = [0, 0, distance + .1]
+        sim.state.set_snapshot(snap)
+
+    def get_energy(simulation):
+        energies = sim.operations.integrator.forces[0].energies
+        if energies is None:
+            return
+        return sum(energies)
+
     r_cut = 2.5
-    r_on = 0.5
-    r = 1.0
+    distance = 1.1
 
-    lj = md.pair.LJ(nlist=md.nlist.Cell(buffer=0.4), default_r_cut=r_cut)
+    lj = md.pair.LJ(nlist=md.nlist.Cell(buffer=0.4), default_r_cut=r_cut * 1.2)
```

Comment:
Took me a second to realize why you'd define r_cut and immediately modify it. But not so long that it needs a comment.